api/src/routes/guilds/#guild_id/index.ts

```diff
@@ -59,6 +59,26 @@ router.patch("/", route({ body: "GuildUpdateSchema"}), async (req: Request, res:
 		where: { id: guild_id },
 		relations: ["emojis", "roles", "stickers"]
 	});
+	
+	if (body.public_updates_channel_id && body.public_updates_channel_id === "1") {
+		const publicUpdatesChannel = await Channel.createChannel({
+			name: Config.get().guild.community.defaultPublicUpdatesChannelName,
+			type: 0,
+			guild_id,
+			permission_overwrites: Config.get().guild.community.defaultPublicUpdatesChannelPermissionOverrides.map(x => ({...x, id: x.id.replace("@everyone", guild_id)}))
```

Comment:
Just a simple code style issue, I'm not a fan of how you repeat `Config.get().guild.community` like, 3 times

Reply:
purely codestyle since we cache queries for 3 seconds, but yeah, please do store that in a variable.

Reply:
I found out recently that the config.get method doesn't actually refresh the cached config, it's only ever fetched on server start when we do Config.init()

maybe should put it on a timer?